rascal-lsp/src/main/rascal/lang/rascal/lsp/refactor/Rename.rsc

```diff
@@ -523,16 +537,19 @@ ProjectFiles preloadFiles(set[loc] workspaceFolders, loc cursorLoc) {
 }
 
 ProjectFiles allWorkspaceFiles(set[loc] workspaceFolders, str cursorName, bool(loc, str) containsName, PathConfig(loc) getPathConfig) {
-    return {
-        // If we do not find any occurrences of the name under the cursor in a module,
-        // we are not interested in loading the model, but we still want to inform the
-        // renaming framework about the existence of the file.
-        <folder, containsName(file, cursorName), file>
-        | folder <- workspaceFolders
-        , PathConfig pcfg := getPathConfig(folder)
-        , srcFolder <- pcfg.srcs
-        , file <- find(srcFolder, "rsc")
-    };
+    ProjectFiles fs = {};
+    for (folder <- workspaceFolders) {
+        pcfg = getPathConfig(folder);
+        for (srcFolder <- pcfg.srcs) {
+            for (loc f <- find(srcFolder, "rsc")) {
```

Comment:
nit: we can inline loops:

```rascal
 for (srcFolder <- pcfg.srcs, loc f <- find(srcFolder, "rsc")) {
```` 

or even:

```rascal
for (folder <- workspaceFolders, srcFolder <- getPathConfig(folder).srcs, loc f <- find(srcFolder, "rsc")) {
```

Reply:
outdated since these changes were reverted